nrrd-design-system/components/components/accordions/accordions.hbs

```diff
@@ -123,7 +123,7 @@
                 <span>
                   Mobile-collapsing item
                 </span>
-                <img class="chart-title-icon" alt="icon with a minus sign" src="/img/icons/icon-circled-minus.svg">
+                <img class="chart-title-icon" alt="icon with a minus sign" src="../../img/icons/icon-circled-minus.svg">
```

Comment:
I'm a little stumped here because the Federalist preview link isn't registering this change, which _should_ be working, but I'm seeing the following still:

![screen shot 2018-04-06 at 3 35 43 pm](https://user-images.githubusercontent.com/11636908/38440700-40780f8c-39b0-11e8-98bd-f4ee180fc8f9.png)


So I can't tell if it's a bug or if you need the format you've used below on the other links, `../../../public/`

Reply:
Hopefully I resolved this one. Thank you!